src/bundle/Tests/Templating/Twig/FieldEditRenderingExtensionTest.php

```diff
@@ -55,6 +55,31 @@ public function getFieldDefinitionData($typeIdentifier, $id = null, $settings =
         ]);
     }
 
+    /**
+     * @dataProvider getLegacyTests
+     * @group legacy
+     *
+     * @param string $file
+     * @param string $message
+     * @param string $condition
+     * @param array $templates
+     * @param string $exception
+     * @param array $outputs
+     * @param string $deprecation
+     */
+    public function testLegacyIntegration(
+        $file,
+        $message,
+        $condition,
+        $templates,
+        $exception,
+        $outputs,
+        $deprecation = ''
+    ): void {
+        // disable Twig legacy integration test to avoid producing risky warning
+        self::markTestSkipped('This package does not contain Twig legacy integration test cases');
+    }
```

Comment:
Method exists in Twig, so has to be declared this way. Looks like a bug on their end (test not asserting anything)